src/sentry/receivers/onboarding.py

```diff
@@ -404,14 +412,18 @@ def record_event_with_first_minified_stack_trace_for_project(project, event, **k
 
     # First, only enter this logic if we've never seen a minified stack trace before
     if not project.flags.has_minified_stack_trace:
+
         # Next, attempt to update the flag, but ONLY if the flag is currently not set.
         # The number of affected rows tells us whether we succeeded or not. If we didn't, then skip sending the event.
         # This guarantees us that this analytics event will only be ever sent once.
         affected = Project.objects.filter(
             id=project.id, flags=F("flags").bitand(~Project.flags.has_minified_stack_trace)
         ).update(flags=F("flags").bitor(Project.flags.has_minified_stack_trace))
 
-        if affected:
+        if (
+            project.date_added > START_DATE_TRACKING_FIRST_EVENT_WITH_MINIFIED_STACK_TRACE_PER_PROJ
```

Comment:
@priscilawebdev seems like this change should warrant a change in tests. Specifically, make sure that we don't send the analytics but do update `has_minified_stack_trace` for older projects.

Reply:
do you mean that we should add a test to verify the new changes? If so, yes I can add a new test

Reply:
@priscilawebdev yes, adding a test to verify the change